ts/components/screens/BaseScreenComponent.tsx

```diff
@@ -324,3 +306,5 @@ export default connectStyle(
   {},
   mapPropsToStyleNames
 )(BaseScreenComponent);
+
+export { handleOnContextualHelpDismissed as test_handleOnContextualHelpDismissed };
```

Comment:
If it is only for test purpose, could this solutions fit your needs?
https://github.com/pagopa/io-app/blob/51e0ff0ed4789304e2944b94a8213f63616c08fd/ts/components/wallet/PayWebViewModal.tsx#L209

Reply:
That's brilliant, thank you! Updated [here](https://github.com/pagopa/io-app/pull/3266/commits/e4e9abf83b7eb1efa3b1a75a66ebf4e12c6fe423#diff-99b6f65a74838b4555bfed0547c21e3f554b6fbca42f36eb9687795a8c2e96b6R308)